src/public/img/about/about-gradient.svg

```diff
@@ -0,0 +1,9 @@
hQcVx1g+MARAIYSX0cF/t0+zMuUDfA1InJJRxnVLA1dOm+vx7mU7JdTKm60Q7XNMXZdRVPdKKG5QKL+xNLzzS8g8CBd3xOjiWLg/XIIfKojBDsj/sMr0v6UDsO9TBnJNMmyUnkQzHZHLYmhwuInZR2LduN322eIW9RLekoEkwnkS3//yrw61KdP9yOG/1mS/9l1Tmnvm9WKmH11lCJh66uZQih6Uu+GZAXtwgsMphaw+XPbz2+GXgqYe7OzdoPkQuJDqBoLitgIH+eMYjgyr1UQ6OXVqGgevT6A7bv5cipVeF7web+q1zuNQbT59u3YizLgSZ3oE4KzObXGPMumLWU6oWq3SAG+/VT4NnEyaVMpfVPQPZF8/8oCZttAA1NKEH8VwqSo9sXF3s5jk2qHt3y4A1xNt9yEnpm6znuNBHzloZbrM6feKWTDVw/Uq26lo0MmbWmiUY+fw+QSj9ZmzvvC2zMSzOLlQNJtWjq7FMjsjzz7/kXqCXoakTEcq4TjDgijVR+a30h8fkgAnwNUQHMCEOrB8tWwK5BAgqvyrHxcGXhbvphUda/kGgoDvdbz0Hl4drEKeyIIRL8snhYiKJ4ftQgyCJ8V29EQOZj+x4ZpUIhaTctTF5XETuorBvbW76bPEKfKluCcETIDyJPu7zrya3K9n6kkQOv24nX4eQc8iQeAbRU2Qc4uoaJB68QS2xxZupe/QAsM1DOaQeacEeYM/Gih5YQrWMXEiUAqiXEztGoP+erlCydcWUA+VumIiOkbUu6sauRVKrqZfncKkFbdheYmYQ0ztujsp6Iy+2Qrkp68HwcXUo1j5ku40Apv8ZHZpx9nP/zjkhNjk7qF/VDo1+Za1swESaz9KHeuXUPPYOQ9kXQHmoOhW8oThNWimavcR3w0cMXN/M/VBQU7OVqAReCMwzS3vRKrwcfgJ9MwkZx0hQ6V3qvaE/E+ZdoJcEqdMPyvgdRXQb6Wh+VsWZGDQJuobpgCbFgrSkY9EONY2gO5RTa7SqRyxyy/K96YVHSv4JIAkQUu/Rc3cB0QUhXK2u/lZ6ogZjqMGniRxcvT2DEJeQ8/o9g6xFUp5efHpmMICK7U2fLV4qF5TrTClIIujuSfQjPv9qdLsyxksiHPTfTgWEkHKojuDaNvw5SceDB9JJtydW02Mv6D3msgJVLvPg6qRxUPQMPOWQvNQOSiGj3xsMKG5bZPJi75SnELRyAf1Ns8ihr+ELUjlOtmH7QjR2PRJr7WnK6OTyV/tL6ClGF6Z/ujmq66/wc/Xb607q0S/keUIz847Rws1tlQ7dpna0qBPAZBokzTwTXO2KnCDbLVWmLyrZgCDM3IBKovTHBgzQrYpPzMjDD2uIdxim/pA+xWww/i+2FSPnyikOKXSnR1wU1cD1G2y6KN8GR/YV/kSCsR0Lv1HB/NV5N+pOAxPm43S9ss5Jqi9zn/Cfv//9P7zPHuKnEP56p2JQxu/YoptPvrFYdVGakE8NcZNLFjov2qEuWNCd7o+Hw6oeacktzfWmFx4pGiSAJEBIvVc6QRwgI53KZHiI2JPDxUSSwo9hbjLjkaLIesiOZ1aJUEjKpk2Xw0XkLgo7LM5tODlVvhXPAJmAq7wOPt3xq9l9K/jn8fT3X4zvm4DCLP+AJF/BiQcusGTqYS8coDHc4uWL+Y7DHrYjqVR8g+TZA3uPXwaem5Jj4PrvSXQMYX/jkI/yLXhIfvsZI+l8Mgf8uyaR2RaZOeLaZdcjudamfuvcaDd7OQrazNSF6R23FcVnhPSDAS4Z44xfWQ/qOlDqDOR0Go85Pv7YUbRTZ7CiHbwAXwoys4N9brolaQxc+8Cz6MAGg7C5DmXvxASpVEWCfQJaVbmQAY0F5CJEDFzfzj4K9mufXuIPIpQwbC+avgYvb9/rbCnP7uqsGxFR1zyQXERXC76bPtc89QX6HmjFARAdg3Z4ZSO6ASWtTY44kwMmwNcQN7loofOiHTr1oiutO6zqkZZhUCouYSPY0vgdtikcgiBBd64OA2uXh2sQp7IgxJDkVWuxJJWkgQ/TsfAd43VmEEJ8Qs5vlkkwTNqmbZfHBQQvCNnXryBzK16ReRvuFgFAFE90Ewj+Dpevnjl1Hr73ci/FFqHPLJN/MB5XJlo+n2M0gu9fcFgmSMJh3SixZx7JyTAH8r7o3OXPKjxXMsfA9d8Tyej7jL4V+1A90FR6SDnsdOsQ0X/zdWM0Tctcop6Wrx9s6uW5H35LVtCGmVVs/5i8O8m4OSrrjVxfshbOvGfW0Kfp4KyVUzppPdEAuboVsx+23ZcV4cCsEaYvoC2Fqgn5t9wrLqA2WCPUQztdH0NEkP6bGDlTTlsOImBnrbgIqoHrV5ZHmcfGRpJ9LrcQBw2K/AuF7S9rHwyFdYwgDCC4jNPulQb7ZORdoFsj+AM/ymMZ/v3v/3F8Irni5ICKQRnXIgRcfTU3iyrG5IAJ8DXETS5aqLwoY7uMCRBwf+yGV13gLcvVCLQs9qAaaRyCQEF3ut8sB4gDZKTTmQwPEXtyuJBIQmgc4iYzflQYPJkOShjg0naokhqmkpAntEAKowtC9ikWZG7FyxH1tlxFYqIb23pfe+sd4weqEnoJDzn/ADEDB5hLs78Dz6QDOEo38fATjRE5zBvFYA7lPa7MBYaH9fg8A88dG1kHkB0cN34rRLR3A334DE0T5r4RuaerHDC71NTZE9CcPaQn1nG/HGqvBfRxtW/6mNGFmUFM37icxbpLublkuB88+sk6nMbQOVm/ZZQ1YvgAZZP+5IPUQ+B8Zp6bbn7qzPhenCNbnKcN0wun56dbwJpGpzeGKG4IZa0N9j1gm9v6BvY/VVw36W748wcEFmTqGEEYs/Q3ZOlvZOAlchgyzz0U3Rr79QX6jnThITt1GEcZv2US3VCdD7sqzsSgCdA1xAAwIRasIx2LdqipBN3hvErDVT3SkluaqxFsafxOZdI4BIGC7lLfSf0wmehkeFenI/6FRIKhcfcLn/FuIQoDPCE
```

Comment:
There's no point in making this svg when it's just a base64 encoded png inside. Use the png directly and delete this.

Reply:
Replaced